### DISCUSSION TITLE
Extension future

### DISCUSSION THREAD

**Discussion by kgshank** [kgshank/gse-sound-output-device-chooser#270] [Announcements]
The next gnome release is around the corner and this time, the extension architecture is undergoing lots of changes.  As I understand the new architecture makes the extension incompatible with gnome version < 45. Due to lack of time and even after the commitments I gave in the past , I couldn't  spend effort in the development.  So as of now compatibility for v43 and v44 are doubtful and I might work on them, if feasible in future. I will start the work for v45 soon.

**Comment by dustin-hawkins** [comment]
I've used the same concepts from your extension to create a newer Gnome 45+ compatible version:

https://github.com/dustin-hawkins/quick-sound-switcher

Its currently pending approval in the extensions shop.
